### PR TITLE
Fix html for meeting and task

### PR DESCRIPTION
**Pull request by guillaumepotier**
- [x] remove html from markdown and plaintext
- [x] replace `<div>` by `<span>`

### CODE REVIEW

**Comment by guillaumepotier** [comment]
Ok for review @krichprollsch :)

File: zap.js [hunk 1]
```diff
@@ -114,9 +115,9 @@ var Zap = {
         for (var i = 0; i < notes.length; i++) {
             switch (notes[i].type) {
                 case "item":
-                    agenda_html += '<h1>' + notes[i].title + '</h1>';
-                    agenda_markdown += '## ' + notes[i].title + "\n";
-                    agenda_plaintext += notes[i].title + "\n";
+                    agenda_html += '<h1>' + this._normalize(notes[i].title) + '</h1>';
+                    agenda_markdown += '## ' + this._sanitize(notes[i].title) + "\n";
```
Comment (krichprollsch) [inline zap.js:119]: an improvement for the future could be to convert this html into valid markdown

Comment (guillaumepotier) [inline zap.js:119]: Sure, thought about it, but was out of scope :)

**Comment by krichprollsch** [comment]
LGTM :+1:

**Pull request merged**